sacro/adapters/zipfile.py

```diff
@@ -0,0 +1,35 @@
+import io
```

Comment:
What is an adapter in this context?

Reply:
Ok, make sense. The word adaptor can have very specific technical meanings in different contexts, especially [in the python ecosystem](https://zopeinterface.readthedocs.io/en/latest/adapter.html).

I think that moving the zipfile code out of the view is good change, whatever the name! :+1: